pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -353,6 +353,13 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 					Placeholder:  "{{ .ExternalURL }}",
 					PropertyName: "client_url",
 				},
+				{ // New in 9.5.
+					Label:        "Details",
+					Description:  "A set of arbitrary key/value pairs that provide further detail about the incident.",
+					Element:      ElementTypeKeyValueMap,
+					InputType:    InputTypeText,
+					PropertyName: "details",
+				},
```

Comment:
I think we need to figure something out here to let users know about the default details and their keys and that they can be overridden.  
Context, 
https://github.com/grafana/alerting/blob/d1e3c68ac0640b20c52411acc30ae36e11624f78/receivers/pagerduty/config.go#L20-L24

```
{
	"firing":       `{{ template "__text_alert_list" .Alerts.Firing }}`,
	"resolved":     `{{ template "__text_alert_list" .Alerts.Resolved }}`,
	"num_firing":   `{{ .Alerts.Firing | len }}`,
	"num_resolved": `{{ .Alerts.Resolved | len }}`,
}
```

However, in the Alertmanager's PagerDuty config, we have the same problem 

![image](https://user-images.githubusercontent.com/25988953/225465411-be9bca5c-70b3-4d14-9769-9d93dca46c89.png)

and therefore I do not think this is a blocker. 

But it would be great to mention something about the default details. Probably refer to the documentation? @brendamuir what do you think?

Reply:
Hi-sure-what do we need to add exactly?

Reply:
Sorry for late reply, I think we need to add an article to the Grafana documentation that mentions PagerDuty integration, its default value for `Details` field see comment above, and then link to that article in the description to the field in UI.